Header header
int16 servo_angle
float32 roadblock_distance
bool stop_bike